### DISCUSSION TITLE
Daily wallpapers

### DISCUSSION THREAD

**Discussion by ghost** [tanujnotes/Olauncher#270] [Q&A]
Where can I manually get daily wallpapers? Maybe there's some site?

**Comment by tanujnotes** [comment]
If you tap on 'Daily new wallpaper' in Olauncher settings, you can download the wallpaper.

**Reply by ghost** [reply]
Isn't there some server or site which all wallpapers come from? To download them all one time.

**Reply by tanujnotes** [reply]
https://gist.github.com/tanujnotes/bf400a269746c5c124a599af040dd82e/ddb25420d15f81719eb93fc8bf532b6addbe403d
